src/styles/base/_variables.scss

```
@@ -3,21 +3,32 @@ $color-success: #48bd66;
 $color-warning: #ffd200;
 $color-danger: #e62a2a;
 $color-info: #5bc0de;
+$background-default: hsl(210, 10%, 96%);
+$palette-dark: rgb(84, 105, 212);
+$palette-light: #9BA6DB;
+$font-default-white: #fff;
```

A few issues with this approach:
* what's the reason for mixing `hsl`, `rgb` and `hex`?
* you started with the prefix `color-`, now there's `palette-` so let's stick to one
* there's only `light` and `dark`, what if you need additional colors?
* you have the `$color-primary` that you're not using. So let's remove `$color-primary: #2461bd;` and add the ones that you have, with lightness scale - 500 is default, each step -100 is lighter, +100 is darker
* `background-` and `font-` prefixes are good but these should be an alias to an existing color. Example:

```suggestion
$color-primary-500: #5469d4;
$color-primary-300: #9BA6DB;

$color-neutral-100: f4f5f6;
$color-white: #fff;

$background-default: $color-neutral-100;
$font-default-white: $color-white;
```